common/build/eslint-config-fluid/src/file.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// This file is used only to run `eslint --print-config`. See the README for this project for more information.
```

Comment:
Did you mean to update the README in this PR?  Instructions for how to use `eslint --print-config` to update the baseline file would be great.

Reply:
Yeah, good catch. I'll update it.